proto/vtrpc.proto

```diff
@@ -59,6 +106,6 @@ enum ErrorCode {
 // We use this so the clients don't have to parse the error messages,
 // but instead can depend on the value of the code.
 message RPCError {
```

Comment:
I don't know yet. I planned on punting on this for now, and thinking about it more carefully when devising a roll out plan, so I don't accidentally do unhappy things to backwards compatibility.